WordPress/src/main/res/layout/new_site_creation_site_info_screen.xml

```diff
@@ -23,32 +23,40 @@
             android:layout_height="wrap_content"
             android:layout_gravity="center_horizontal">
 
-            <TextView
```

Comment:
This looks like a big change, but I just wrapped title and subtitle with a linear layout.